e2e/test/scripts/infrastructure-linode/0.1.0/cluster-template-clusterclass-kubeadm.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
These files in `infrastructure-linode/*` should all be available as release artifacts once we have a release(hopefully this week)

Reply:
CAPL has it's first release so these should be able to be removed at this point